docs/python/debugging.md

```diff
@@ -171,7 +191,7 @@ An array of additional options that may contain the following:
 | `"DebugStdLib"` | Enabled debugging of standard library functions. |
 | `"Django"` | Activates debugging features specific to the Django web framework. |
 | `"Sudo"` | When used with `"console": "externalTerminal"`, allows for debugging apps that require elevation. Using an external console is necessary to capture the password. |
-| `"Pyramid"` | Used when debugging a Pyramid application. |
+| `"Pyramid"` | When set to true, ensures that a Pyramid app is launched with [the necessary pserve command](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/startup.html?highlight=pserve). |
```

Comment:
I would make `pserve` be marked as code as it's a specific command.

Reply:
Will do.